src/promptflow/promptflow/_sdk/schemas/_connection.py

```diff
@@ -111,3 +111,42 @@ class CustomConnectionSchema(ConnectionSchema):
     configs = fields.Dict(keys=fields.Str(), values=fields.Str())
     # Secrets is a must-have field for CustomConnection
     secrets = fields.Dict(keys=fields.Str(), values=fields.Str(), required=True)
+
+
+class CustomStrongTypeConnectionSchema(YamlFileSchema):
```

Comment:
Why not implement the CustomConnectionSchema class?

Reply:
fixed